design/README.md

```diff
@@ -0,0 +1,53 @@
+# Obscuro design documents
+
+Obscuro started based on a (whitepaper)[https://whitepaper.obscu.ro/] published in Nov. 2021.
+
+As we started development, we started thinking more deeply about some aspects and also getting feedback from the community
+and the users.
+This folder contains design proposals which either replace some aspects of the whitepaper or add more detail.
+
+## Architecture
+
+Useful resources to understand the system.
+
+
+## User experience
+
+UX was an area that we didn't cover extensively in the WP. So we focused on it early during development, and our approach is documented here.
+
+The design documents cover aspects like:
+1. The wallet extension
+2. The data relevancy
+3. The incentives for node operators to provide services
+
+Note: the wallet extension and the data relevancy rules are already implemented. 
+
+
+## Bridge
+
+Obscuro needs a decentralised bridge infrastructure to Ethereum.
+
+Note: this design is approved and implemented.
```

Comment:
Perhaps extend the note to say that while this is implemented, the security of it depends on the fast finality being finished.